core/scheduler/handler/v1beta1/replay.go

```diff
@@ -63,6 +65,36 @@ func (h ReplayHandler) Replay(ctx context.Context, req *pb.ReplayRequest) (*pb.R
 	return &pb.ReplayResponse{Id: replayID.String()}, nil
 }
 
+func (h ReplayHandler) ListReplay(ctx context.Context, req *pb.ListReplayRequest) (*pb.ListReplayResponse, error) {
```

Comment:
should we limit the list of Replay to be shown, for example only the latest 3 months Replay?

Reply:
If that so, we can add another param `dayLimits` maybe?

Reply:
sure that also works.
for now, I think we can just have a default value to limit the list of Replay and later can expose a param through CLI flag when it is needed. WDYT?

Reply:
yes, agree